.circleci/config.yml

```diff
@@ -4,62 +4,97 @@
 #
 version: 2
 jobs:
-  build:
+  mysql:
     docker:
       # specify the version you desire here
-      - image: circleci/ruby:2.5.1-node-browsers
-
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
+      - image: circleci/ruby:2.5-node-browsers
```

Comment:
This seems like a very unusual change. Does this match with the image we use on any other repo?

Reply:
now using the original version